### PR TITLE
Restrict servicerating use to clinic registered contacts

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

File: go-app-chw.js [hunk 1]
```diff
@@ -65,8 +65,6 @@ go.utils = {
     is_out_of_hours: function(config) {
         var today = go.utils.get_today(config);
         var motoday = moment.utc(today);
-        console.log(motoday);
```
Comment (gsvr) [inline go-app-chw.js:68]: This seems to have been left over from a previous commit that hadn't been compiled before committing.

File: src/servicerating.js [hunk 2]
```diff
@@ -126,6 +134,13 @@ go.app = function() {
             });
         });
 
+        self.states.add('end_reg_clinic', function(name) {
+            return new EndState(name, {
+                text: 'Please register at a clinic before using this line.',
```
Comment (gsvr) [inline src/servicerating.js]: I purposefully left this untranslated, let me know if we should translate it.

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/servicerating.js (hunk 2 above)
Comment (imsickofmaps) [inline src/servicerating.js]: Please mark for translation, I'll raise when closing JIRA

**Comment by imsickofmaps** [comment]
Minor change then :+1:  and land

**Pull request merged**